synphot/expose.py

```diff
@@ -0,0 +1,439 @@
+        Note: This radiance component represents an instrumental effect.
+        The emission is provided relative to the other model components.
+        To obtain the correct absolute flux, an instrumental response curve
+        must be applied to the resulting model spectrum.
+        See section 6.2.4 in the SkyCalc documentation at
+        http://localhost/observing/etc/doc/skycalc/
+        The_Cerro_Paranal_Advanced_Sky_Model.pdf
+    therm_t1, therm_t2, therm_t3 : float
+        Temperature in K (default is 0.0)
+    therm_e1, therm_e2, therm_e3: float
+        In range [0,1] (default is 0.0)
+
+    vacair: str
+        In regards to the wavelength grid.
+        options: ['vac', 'air] (default is 'vac')
+    wmin: float
+        Minimum wavelength (nm) in the wavelength grid.
+        Must be in range [300.0,30000.0] and < wmax
+        (default is 300.0)
+    wmax: float
+        Maximum wavelength (nm) in the wavelength grid.
+        Must be in range [300.0,30000.0] and > wmin
+        (default is 2000.0)
+    wgrid_mode: str
+        Mode of the wavelength grid.
+        options: ['fixed_spectral_resolution','fixed_wavelength_step', 'user']
+        (default is 'fixed_wavelength_step')
+    wdelta: float
+        Wavelength sampling step dlam in range [0,30000.0] (nm/step)
+        (default is 0.1)
+    wres: int
+        lam/dlam where dlam is wavelength step.
+        In range [0,1.0e6] (default is 20000)
+    wgrid_user: list of floats
+        default is [500.0, 510.0, 520.0, 530.0, 540.0, 550.0]
+
+    lsf_type: str
+        Line spread function type for convolution.
+        options: ['none','Gaussian','Boxcar'] (default is 'none')
+    lsf_gauss_fwhm: float
+        Gaussian full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+    lsf_boxcar_fwhm: float
+        Boxcar full-width half-max for line spread function wavelength bins.
+        Range > 0.0 (default is 5.0)
+
+    observatory: str
+        Observatory where observation takes place.
+        Options are 'paranal', 'lasilla', 'armazones' (default is 'paranal')
+
+    Returns
+    -------
+    trans_waves, transmission: tuple of arrays of floats
+        'trans_waves' is an array of wavelengths in angstroms (float),
+        'transmission' is an array of fractional atmospheric
+        transmittance (float).
+
+    """
+
+    params = locals()
+
+    if params['observatory'] == 'lasilla':
+        params['observatory'] = '2400'
+    elif params['observatory'] == 'paranal':
+        params['observatory'] = '2640'
+    elif (params['observatory'] == '3060m' or
+          params['observatory'] == 'armazones'):
+        params['observatory'] = '3060'
+    else:
+        raise ValueError('Wrong Observatory name, please refer to the '
+                         'skycalc_cli documentation.')
+
+    # Use the bit from skycalc_cli which queries from the SkyCalc Sky Model
+    server = 'http://etimecalret-001.eso.org'
```

Comment:
Why is this not an `astroquery` call?

Reply:
That bit is an artifact from me borrowing straight from skycalc_cli. Is there an astropy equivalent to `requests`?

Reply:
There is a `astropy.utils.data.download_file()` function but I think @bmorris3 said that it is not quite what you need. So my other suggestion would be to move the skycalc query out to its own module (after you created this "experimental" folder and move your work there).